### ISSUE TITLE
[rfe] option to hide horizontal panel

### ISSUE THREAD

**Issue by juhp** [material-shell/material-shell#738]
**Is your feature request related to a problem? Please describe.**

The horizontal panel (like the original one in gnome-shell) takes up quite a lot of desktop real-estate and for me most of it is just empty space... so it seems kind of wasteful.

**Describe the solution you'd like**

It would be great if there was a config option to turn of the horizontal panel.
(It would probably move the layout popup menu button over to the vertical panel.)

**Describe alternatives you've considered**

It could also optionally merge the horizontal panel widgets into to the vertical one but that would be a stretch goal/idea perhaps.
(but not sure if that is necessary or even desirable).

_ps_
I just discovered material-shell yesterday and am totally blown away by! Very cool indeed, thank you!!

**Comment by Sebastian-Jung** [comment]
Hey @juhp,
Maybe take a look at the `Zen-Mode`, it's already available using the keyboard shortcut `Super+ESC`. It will hide the horizontal panel AND the side panel, similar to a full screen. Currently, I don't think there is an option to only hide the horizontal panel, but this might get close to what you are looking for.

**Comment by juhp** [comment]
Yes I saw the Zen mode, but I really want the vertical panel but not the horizontal one.

**Comment by juhp** [comment]
Anyway for now I just switched back to my vertical dash-to-panel. :)

I generally only have one app per workspace, so the window list in the horizonal panel here is not useful to me.

**Comment by wojpawlik** [comment]
In half mode, all windows are visible, rendering taskbar redundant.

**Comment by TunaBicim** [comment]
Is there any plans to work on this? @wojpawlik is correct that half mode has no use for the taskbar. Alternatively a way to set the size of vertical panel different from horizontal panel so it can be made smaller.
